app/controllers/posts_controller.rb

```diff
@@ -10,4 +10,28 @@ def show
     @comments = @post.comments.all
     @likes = @post.likes.all
   end
+
+  def new
+    @current = current_user
+  end
```

Comment:
- [x] OPTIONAL - unused variable inside the new method `@current`, I suggest you initialize a new post like you did in the comments controller and use the variable inside of the create method